http/src/main/scala/org/http4s/blaze/http/http2/FrameSerializer.scala

```diff
@@ -305,7 +305,7 @@ private[http2] object FrameSerializer {
       length: Int,
       frameType: Byte,
       flags: Byte,
-      streamdId: Int,
+      streamId: Int,
```

Comment:
?

Reply:
This definitely shouldn't be changed

Reply:
Is this a type-signature problem? The rest of the file definitely seems to be thinking about `stream id` or `streamId`.